### PR TITLE
Added my adviser Dr. Serghei Mangul as a coauthor.

### PR DESCRIPTION
**Pull request by Sergey-Knyazev**
Dr. Mangul doesn't use GitHub, so he asks me to introduce his changes.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
Hi @Sergey-Knyazev, would you mind if I reach out on email to him and confirm this is correct? We just like to have the author official sign off on these entries (and have a record of it), especially for the code of conduct and conflict of interest. 

Also, can I ask whether he primarily wrote new text or edited what you wrote?

Thank you so much for doing this!

**Comment by Sergey-Knyazev** [comment]
> Hi @Sergey-Knyazev, would you mind if I reach out on email to him and confirm this is correct? We just like to have the author official sign off on these entries (and have a record of it), especially for the code of conduct and conflict of interest.
> 
> Also, can I ask whether he primarily wrote new text or edited what you wrote?
> 
> Thank you so much for doing this!

Hi, Rando,
Sure, please email him. Initially, I wrote the text send it to review to Serghei, then Serghei reviewed it, commented, and proposed changes.

File: content/metadata.yaml [hunk 1]
```diff
@@ -630,3 +630,17 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-08-03
+  -
+    github: smangul1
+    name: Serghei Mangul
+    initials: SM
+    orcid: 0000-0003-4770-3443
+    twitter: serghei_mangul
+    email: serghei.mangul@gmail.com
```
Comment (rando2) [inline content/metadata.yaml:639]:
```suggestion
    email: serghei.mangul@gmail.com
    contributions:
      - Writing - Review & Editing
```

**Comment by rando2** [comment]
Approved by email on 8/10/20. Thanks again for doing this, @Sergey-Knyazev!

**Comment by rando2** [comment]
Appveyor failed but artifact looks fine

**Pull request merged**